Wikipedia/Code/NSAttributedString+WMFSavedPagesAttributedStrings.m

```diff
@@ -2,14 +2,11 @@
 #import "NSString+FormattedAttributedString.h"
 @import WMF;
 
-static NSInteger const kTitleColor = 0x000000;
```

Comment:
Actually I think you might be able to remove this and its header file entirely, it doesn't look like it's used anymore in the app.